src/basic/selinux-util.c

```diff
@@ -35,14 +35,14 @@ DEFINE_TRIVIAL_CLEANUP_FUNC(context_t, context_free);
 static int mac_selinux_reload(int seqno);
 
 static int cached_use = -1;
-static int cached_enforcing = -1;
 static struct selabel_handle *label_hnd = NULL;
 
-#define log_enforcing(...) log_full(mac_selinux_enforcing() ? LOG_ERR : LOG_WARNING, __VA_ARGS__)
+#define log_enforcing(...)                                              \
+        log_full(security_getenforce() > 0 ? LOG_ERR : LOG_WARNING, __VA_ARGS__)
```

Comment:
I prefer `security_getenforce() != 0`.

[`security_getenforce()`](https://github.com/SELinuxProject/selinux/blob/master/libselinux/src/getenforce.c) returns `1` on erforced mode, `0` on permissive mode and `-1` on error.
The error cases are:
* `/sys/fs/selinux` is not mounted, mainly because SELinux is disabled.
  In the systemd code we always assure SELinux is enabled prior: either via `mac_selinux_use()` or `label != NULL` (which is only set if SELinux is enabled), so this error should never occur (or means a bug in the systemd code path).
* `/sys/fs/selinux/enforce` can not be opened.
  Since this is a public inode (`-rw-r--r--. 1 root root 0 Jan  1  1970 /sys/fs/selinux/enforce`) pid1 and systemd user instances should have access, so this should also never happen.
* Failed to `read()` or convert (`sscanf()`) the content of `/sys/fs/selinux/enforce`

I think in case of any unlikely failure, using `LOG_ERR` above `LOG_WARNING` is sensible.